code/game/objects/items/weapons/circuitboards/circuitboard.dm

```diff
@@ -6,24 +6,37 @@
 
 /obj/item/circuitboard
 	name = "circuit board"
-	desc = "Looks like a circuit. Probably is."
+	desc = "A circuitboard, an electronic device which forms the backbone of most modern machinery."
 	icon = 'icons/obj/module.dmi'
 	icon_state = "id_mod"
 	item_state = "electronic"
 	origin_tech = list(TECH_DATA = 2)
-	density = 0
-	anchored = 0
-	w_class = 2.0
+	w_class = ITEMSIZE_SMALL
 	flags = CONDUCT
-	force = 5.0
-	throwforce = 5.0
+	force = 5
+	throwforce = 5
 	throw_speed = 3
 	throw_range = 15
-	var/build_path = null
+	var/build_path
 	var/board_type = "computer"
-	var/list/req_components = null
+	var/list/req_components
 	var/contain_parts = 1
 
+/obj/item/circuitboard/examine(mob/user)
+	..()
+	if(build_path)
+		var/obj/machine = new build_path // instantiate to get the name and desc
+		to_chat(user, SPAN_NOTICE("This circuitboard will build: <b>[capitalize_first_letters(machine.name)]</b>"))
```

Comment:
```suggestion
		to_chat(user, SPAN_NOTICE("This circuitboard will build a <b>[capitalize_first_letters(machine.name)]</b>."))
```
this will probably look better